examples/2d/elixir_euler_density_wave.jl

```diff
@@ -0,0 +1,57 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the compressible Euler equations
+gamma = 1.4
+equations = CompressibleEulerEquations2D(gamma)
+
+initial_conditions = initial_conditions_density_wave
+
+surface_flux = flux_central
+#volume_flux  = flux_central
+polydeg = 5
+#solver = DGSEM(polydeg, surface_flux, VolumeIntegralFluxDifferencing(volume_flux))
+solver = DGSEM(polydeg, surface_flux)
```

Comment:
```suggestion
surface_flux = flux_central
polydeg = 5
solver = DGSEM(polydeg, surface_flux)
```
I think we don't need these options as comments here.